src/types/global.d.ts

```diff
@@ -44,6 +41,7 @@ type Guest = {
     vegan?: boolean;
```

Comment:
Are there two vegan on the same type now? 
Should the "?" not be removed now on both vegan and vegetarian?